src/contents/en-gb/doc_web.md

```diff
@@ -79,62 +65,69 @@ let appInfo = {
   version: '0.1.0',
   vendor: 'MaidSafe.net Ltd.'
 };
-let safeAppHandle = await window.safeApp.initialise(appInfo);
+const safeApp = await window.safe.initialiseApp(appInfo);
```

Comment:
just for consistency with the complete code we share further down don't make it a `const`